src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -106,15 +110,96 @@ limitations under the License.
     <item fieldId="2" name="CoolSetpoint" type="int16s" isNullable="true"/>
   </struct>
 
+  <bitmap name="PresetTypeFeaturesBitmap" type="bitmap8">
```

Comment:
Spec says this is derived from map16, no?

Reply:
I'll have to regenerate my version of the spec and check.  I made mine a while a go, and have been working off that until something was finalized.